src/main/java/io/lettuce/core/resource/DefaultClientResources.java

```diff
@@ -442,7 +442,7 @@ public Builder computationThreadPoolSize(int computationThreadPoolSize) {
 
         /**
          * Sets the {@link DnsResolver} that is used to resolve hostnames to {@link java.net.InetAddress}. Defaults to
-         * {@link DnsResolvers#JVM_DEFAULT}
+         * {@link DnsResolvers#UNRESOLVED}
```

Comment:
see line 306 in this file, this comment was leaving unchanged since the code behavior has changed.

Reply:
Good catch, thanks